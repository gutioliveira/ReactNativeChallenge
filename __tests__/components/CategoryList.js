import 'react-native';
import React from 'react';
import CategoryList from '../../app/components/categoryList';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {

    const tree = renderer.create(
        <CategoryList category={[{id: 0, title: 'hue'}]} onPress={()=>{}}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});