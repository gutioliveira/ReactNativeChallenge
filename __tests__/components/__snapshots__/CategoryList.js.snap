// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexDirection": "row",
        "marginBottom": 2,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 2,
        "width": 750,
      }
    }
    testID={undefined}
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flexDirection": "row",
        }
      }
      testID={undefined}
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderColor": "#9eacb4",
              "borderRadius": 12,
              "borderWidth": 2,
              "height": 24,
              "justifyContent": "center",
              "width": 24,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "marginLeft": 10,
          }
        }
      >
        hue
      </Text>
    </View>
  </View>
</View>
`;
